README.md

```diff
@@ -95,7 +95,8 @@ Assuming the app is bound to a Conjur service instance, pushing the app will res
 
```

Comment:
The README needs to be updated to:
- Update the instructions for uploading to CF from `git clone` to include the package step
- Add instructions for uploading to CF from downloading a ZIP in the release